buildconfig/CMake/PythonPackageTargetFunctions.cmake

```diff
@@ -93,8 +94,17 @@ CustomInstallLib = patch_setuptools_command('install_lib')
         # On UNIX systems install the workbench executable directly.
         # The Windows case is handled with a custom startup script installed in WindowsNSIS
         if ( NOT WIN32 )
-          install(PROGRAMS ${_setup_py_build_root}/install/bin/${_executable_name} DESTINATION bin)
+            install(PROGRAMS ${_setup_py_build_root}/install/bin/${pkg_name}
+                DESTINATION ${WORKBENCH_BIN_DIR}
+                RENAME launch_workbench)
```

Comment:
The renaming here has broken the Linux packaging.  Let's remove the `RENAME launch_workbench` and leave it as workbench. We'll then need to update [MantidWorkbench_osx_launcher](https://github.com/samueljackson92/mantid/blob/workbench-package-osx/buildconfig/CMake/Packaging/osx/MantidWorkbench_osx_launcher#L6) to change `./launch_workbench` to just `./workbench`.